packages/graphback-cli/src/commands/init.ts

```diff
@@ -0,0 +1,62 @@
+import * as execa from 'execa'
+import { accessSync, mkdirSync } from 'fs'
+import { prompt as ask } from 'inquirer'
+import ora from 'ora'
+import * as yargs from 'yargs'
+import { extractTemplate } from '../helpers/extractTemplate'
+import { allTemplates, Template } from '../helpers/template'
+import { logError, logInfo } from '../utils'
+
+type Params = { name?: string }
+
+
+async function installDependencies(name: string): Promise<void> {
```

Comment:
Move those methods to separate file like utils or something. 
Let's keep the right architecture and actual command file clear so anyone can come and see straightaway what are the inputs and execution logic.